src/scss/_theme.scss

```diff
@@ -33,26 +32,26 @@ $input-padding: $spacing-base * 1.5;
 $boolean-padding: $spacing-double;
 
 :root, .apos-theme-light {
-  --primary: #6516DD;
-  --primary-hover: #5007C0;
-  --primary-active: #4000A1;
-  --primary-disabled: #C5A5F4;
+  --primary-button: #6516DD;
+  --primary-button-hover: #5007C0;
+  --primary-button-active: #4000A1;
+  --primary-button-disabled: #C5A5F4;
 
-  --secondary: #B327BF;
-  --secondary-hover: #A107C0;
-  --secondary-active: #8804A3;
-  --secondary-disabled: #F4A5CD;
+  --default-button: #F3F3F3;
+  --default-button-hover: #E2E2E2;
+  --default-button-active: #CDCDCD;
+  --default-button-disabled: #FAFAFA;
```

Comment:
Looking a bit more closely at this, the default button seems to be tied to light/dark theme neutrals, rather than its own, dedicated color. Is that right?

<img width="886" alt="Screen Shot 2020-03-11 at 4 02 00 PM" src="https://user-images.githubusercontent.com/2650558/76472164-f7719a00-63b1-11ea-9e91-7d11432dbdf9.png">

Reply:
will try this, would be great to have default ui generated by neutrals yes

Reply:
this is all generated by neutrals now